packages/common-cosmos/src/project/versioned/v1_0_0/model.ts

```diff
@@ -72,13 +63,17 @@ export class CosmosProjectNetworkDeployment {
   @IsOptional()
   @IsArray()
   bypassBlocks?: (number | string)[];
+  @IsOptional()
+  @Type(() => CosmosCustomModuleImpl)
+  @ValidateNested({each: true})
+  chainTypes?: Map<string, CosmosCustomModuleImpl>;
 }
 
-export class CosmosProjectNetwork extends CommonProjectNetworkV1_0_0<FileType> {
+export class CosmosProjectNetwork extends CommonProjectNetworkV1_0_0<Map<string, CosmosCustomModuleImpl>> {
```

Comment:
I think `Map<string, CosmosCustomModuleImpl>` can be pulled into its own type and used in common-cosmos